exist-core/src/main/java/org/exist/xquery/pragmas/Optimize.java

```diff
@@ -39,15 +39,17 @@
 import java.util.Iterator;
 import java.util.List;
 
-public class Optimize extends Pragma {
+import static java.lang.System.arraycopy;
 
-    public  final static QName OPTIMIZE_PRAGMA = new QName("optimize", Namespaces.EXIST_NS, "exist");
+public class Optimize extends Pragma {
+    public final static String LOCAL_NAME = "optimize";
```

Comment:
See my comment above about why renaming this to `LOCAL_NAME` is problematic and needs to be reversed.